inc/targetproblem.class.php

```diff
@@ -669,6 +558,58 @@ public static function countItemsToImport(array $input) : int {
       return 1 + self::countChildren($subItems, $input);
    }
 
+   public function defineTabs($options = []) {
```

Comment:
like target ticket and target change : swtch to a multiple tabs design to edit this target